node/pkg/watchers/sui/watcher.go

```diff
@@ -399,11 +399,14 @@ func (e *Watcher) Run(ctx context.Context) error {
 
 			}
 
+			// curl -s -X POST -d '{"jsonrpc":"2.0", "id": 1, "method": "sui_getLatestCheckpointSequenceNumber", "params": []}' -H 'Content-Type: application/json' http://127.0.0.1:9000
 			// Epoch is currently not ticking in 0.16.0.  They also
 			// might release another API that gives a
 			// proper block height as we traditionally
 			// understand it...
 			currentSuiHeight.Set(float64(res.Result.Epoch))
+			logger.Error(fmt.Sprintf("sui_getCommitteeInfo: %d", res.Result.Epoch))
```

Comment:
Don't need to do a sprintf here. That being said, @panoel is reworking this whole file under a separate PR, so it will be rereviewed then.